LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Is MIT the good license for us? It allows commercial use from others, and I think we prefer to keep that right for us only.

What about this creative commons license: CC BY-NC-SA
https://creativecommons.org/licenses/by-nc-sa/4.0/

Reply:
👍

Reply:
Tu peux la refuser: on va refaire une PR avec CC BY-NC-SA... 
J'ai pas trouvé le bouton pour supprimer la PR